### PR TITLE
Events, props, and player animations

### PR DESCRIPTION
**Pull request by leo-jh-li**
- player spine animations
- basic inventory system and UI. player can see their currently held prop and also drop their prop (via temporary button)
- destroyed props appear destroyed visually
- event interaction finished. an event needs a certain number of players with the proper props to gather and press repair (or sabotage) to begin event progress
- minor merge fixes
  - removed obsolete Props and Event layers
  - set new Player prefab to Player layer
  - props and car updated to use LookAtCamera instead of Billboard
  - car prefab positioning fixed

### CODE REVIEW

File: DeceptionGame/Assets/Scripts/EventManager.cs [hunk 1]
```diff
@@ -3,64 +3,200 @@
 using UnityEngine;
 
 public class EventManager : InteractableObject {
```
Comment (antu3199) [inline DeceptionGame/Assets/Scripts/EventManager.cs:5]: Not sure how I feel about the EventManager being an "Interactable Object" :/ I think the interactable logic and the state should be separated

File: DeceptionGame/Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -1,23 +1,23 @@
-﻿using System.Collections;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerController : MonoBehaviour {
     [Header("References")]
     public ShowOnlyIfInRange showIfInRangeScript;
```
Comment (antu3199) [inline DeceptionGame/Assets/Scripts/PlayerController.cs:7]: Doesn't have to be now, but I think that animation and prop logic should be separated into their own classes, with PlayerController simply having a reference to them (e.g. like PlayerMovementController)

File: DeceptionGame/Assets/Scripts/UI/EventUI.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections;
+using System.Collections.Generic;
```
Comment (antu3199) [inline DeceptionGame/Assets/Scripts/UI/EventUI.cs:2]: TODO: Should tie into my manager/controller system on merge

**Review by antu3199 — APPROVED** [review]

File: DeceptionGame/Assets/Scripts/EventManager.cs (hunk 1 above)
Comment (leo-jh-li) [reply DeceptionGame/Assets/Scripts/EventManager.cs:5]: big agree

**Pull request merged**